specifications/default-tracks.org

```diff
@@ -37,14 +37,14 @@ With that criteria, this is how the output shall display for a given snap:
 Setting a default track shall have an explicit command to do so.
 
 #+BEGIN_SRC sh
-$ snapcraft set-default-track <snap-name> <revision>
+$ snapcraft set-default-track <snap-name> <track>
 #+END_SRC
 
 When setting a valid track, this is what the command shall output:
 
 #+BEGIN_SRC sh
 $ snapcraft set-default-track node 17
-Default track for node has been set to 17
+Default track for 'node' set to '17'.
```

Comment:
I find in these cases it can be nice to see the prior value, in case you want to restore it. e.g.:
Default track for 'node' was updated from '16' to '17'.

Reply:
what is the difference from just checking the track list?

Reply:
This would be quite useful, btw. There's no harm in printing the previous track, and it can give devs the ability to see what they had (can be useful if you automate the functionality and have a log in a ci/cd system). For that matter, track-revert would also be a nice option to go back to whatever default-1  was.